src/vstart.sh

```diff
@@ -1017,7 +1017,8 @@ do_rgw()
     i=0
     for rgw in j k l m n o p q r s t u v; do
         echo start rgw on http${CEPH_RGW_HTTPS}://localhost:$((CEPH_RGW_PORT_NUM + i))
-        run 'rgw' $RGWSUDO $CEPH_BIN/radosgw -c $conf_fn --log-file=${CEPH_OUT_DIR}/rgw.$rgw.log ${RGWDEBUG} --debug-ms=1 -n client.rgw "--rgw_frontends=${rgw_frontend} port=$((CEPH_RGW_PORT_NUM + i))${CEPH_RGW_HTTPS}"
+        current_port=$((CEPH_RGW_PORT_NUM + i))
+        run 'rgw' $RGWSUDO $CEPH_BIN/radosgw -c $conf_fn --log-file=${CEPH_OUT_DIR}/radosgw.${current_port}.log --admin-socket=${CEPH_OUT_DIR}/radosgw.${current_port}.asok --pid-file=${CEPH_OUT_DIR}/radosgw.${current_port}.pid ${RGWDEBUG} --debug-ms=1 -n client.rgw "--rgw_frontends=${rgw_frontend} port=${current_port}${CEPH_RGW_HTTPS}"
```

Comment:
> Use rgw port instead of meaningless 'j k' in the file name

why do you think the port # is better than the j, k, etc. IMO, this follows the convention we are using in other daemons, like OSD and mon.

Reply:
The difference between rgw and other daemon is we need to specify the `port_num` when testing rgw via misc http client, like curl.  Figuring out auto-assigned `port_num` is boring.

Reply:
makes sense to me!